app/code/Magento/CatalogGraphQl/Model/AttributesJoiner.php

```diff
@@ -78,6 +78,7 @@ public function getQueryFields(FieldNode $fieldNode, ResolveInfo $resolveInfo):
 
                     foreach ($spreadFragmentNode->selectionSet->selections as $spreadNode) {
```

Comment:
is there a guarantee for `->selectionSet`->selections  and `->name->value` ? Nodes in webonyx are pretty complex

Reply:
I added `$selectedFields[] = $spreadNode->name->value;` because in below query `price_range` field was ignored
```
price_range {
    minimum_price {
      final_price {
        value
      }
    }
  }
```